<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca Dipendente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_dipendente">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
